staff/francisco-fernandez/cv-styled/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Curriculum vitae de Patricia Medina</title>
+</head>
+
+<body>
+    <header>
+        <h1>CV Patricia Medina López</h1>
+    </header>
+    <section class="grid-container">
+        <div class="item1 leftcontainer">
```

Comment:
better use "__" for BEM methodology?

Reply:
Yes but it was not my intention to use BEM, i used two classes in the div. Its not a good way of work, i will change it in the future